src/observer/consume.py

```diff
@@ -87,21 +87,22 @@ def do_safely(row):
 # generic content consumption
 #
 
-# this whole function is a bit sucky
 def content_type_from_endpoint(endpoint):
+    """given an API `endpoint` like `press-packages/{id}`, returns a slugified version like `press-packages-id`.
+    if a given API `endpoint` doesn't exist, it's value is looked up in a map of aliases.
```

Comment:
"if the given API `endpoint` doesn't exist in the alias map, it's value is returned as-is."